config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "contracts": {
+    "0x852deD011285fe67063a08005c71a85690503Cee": "OssifiableProxy",
```

Comment:
Can we add the comments what are the proxies here?

Reply:
There is no comments in json. 🤷